public/sw.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable */
+var version = 'v0.1';
```

Comment:
Can we make this match the version on `package.json`? For sanity reasons mainly.